docs/usability/ipfs-webui-useabilty-test.md

```diff
@@ -1,82 +1,49 @@
-# IPFS Web UI Usability Testing
+# IPFS Web UI Usability Test
 
-The app should make the following actions **easy** to complete
+> Visit webui.ipfs.io and find the peer ID of your node
 
-- Add a file and share it with a friend
-- See the files in your repository
-- Determine if the node is connected to the network and operating normally
-- Diagnose when your IPFS daemon isn't running and fix it
-- Find the peer ID of the node you are connected to
-- Find IPLD info about an IPFS path and browse a link
-- Discover and try out experimental IPFS features _missing_ [ipfs-webui#814](https://github.com/ipfs-shipyard/ipfs-webui/issues/814)
-- Find the total size of your IPFS repo _missing_
+  - _Do they have a daemon running already?_
+  - _Do they follow the prompts to start their daemon?_
+  - _Do they have to go and download go-ipfs first?_
 
-More holistically, it should give a feeling of being connected to a network of peers.
-It must not give a sense of "uploading to the cloud", as that is a significant cause
-of confusion about how IPFS works, and it's dangerously misleading.
+> Add a picture you'd like to share to your IPFS node
```

Comment:
What if we change this to (or add it as a separate item):
- Add two files/pictures and share then with a friend

This could produce more data, for example: Are people sending one by one and sending two links, or do they notice that multiple files could be selected and shared together? Are our GUIs communicating the fact multiple files can be bundled and shared together?

We could ask follow-up questions about content-addressing and the meaning of root CID (wrapping directory) etc.  (I suspect having more than one file make it easier to reason about wrapping directory and related concepts)